ddtrace/contrib/aioredis/patch.py

```diff
@@ -120,6 +121,7 @@ def traced_13_execute_command(func, instance, args, kwargs):
         child_of=parent,
     )
     span.set_tag_str(COMPONENT, config.aioredis.integration_name)
+    span.set_tag(db.SYSTEM, redisx.APP)
```

Comment:
If `redisx.APP` is a text type, can we replace this with the more efficient `span.set_tag_str` method?
```suggestion
    span.set_tag_str(db.SYSTEM, redisx.APP)
```